### PR TITLE
FIX: #790 -  Remove single user icon from containerHeader

### PR DESCRIPTION
**Pull request by AllanOcelot**
- Added active state to the users icon
- Removed the user (single) icon from containerHeader

### CODE REVIEW

File: src/components/ContainerHeader.vue [hunk 1]
```diff
@@ -29,17 +29,10 @@
...
-                >
-                    <a @click="sidebarState.close()">
-                        <i class="fa fa-user" aria-hidden="true"/>
-                    </a>
-                </div>
                 <div
                     :class="{
-                        'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist'
+                        'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist',
+                        'kiwi-header-option--active': sidebarState.sidebarSection === 'user'
```
Comment (prawnsalad) [inline src/components/ContainerHeader.vue]: You can't have multiple same keys in an object so this should be something like `'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist' || sidebarState.sidebarSection === 'user'`

File: src/components/ContainerHeader.vue [hunk 2]
```diff
@@ -278,6 +271,13 @@ export default {
                 network.ircClient.join(channelName);
             }
         },
+        nicklistToggle: function nicklistToggle() {
+            if (this.sidebarState.sidebarSection === 'user') {
+                this.sidebarState.close();
```
Comment (prawnsalad) [inline src/components/ContainerHeader.vue]: If it's currently on the userbox then it go back to the nicklist, not close the whole sidebar

File: src/components/ContainerHeader.vue [hunk 3]
```diff
@@ -278,6 +271,13 @@ export default {
                 network.ircClient.join(channelName);
             }
         },
+        nicklistToggle: function nicklistToggle() {
```
Comment (prawnsalad) [inline src/components/ContainerHeader.vue]: This should use the function shorthand, `nicklistToggle() {`. Also lets rename it to `toggleNicklist()` so it's consistent with the function name a few lines below

**Comment by AllanOcelot** [comment]
@prawnsalad  these have all been fixed, please review when you get the chance :)

File: src/components/ContainerHeader.vue [hunk 4]
```diff
@@ -29,23 +29,16 @@
...
                     :class="{
                         'kiwi-header-option--active': sidebarState.sidebarSection === 'nicklist'
+                            || sidebarState.sidebarSection === 'user'
                     }"
                     class="kiwi-header-option kiwi-header-option-nicklist"
                 >
                     <a
                         :title="$t('person', {count: Object.keys(buffer.users).length})"
-                        @click="sidebarState.toggleNicklist()"
+                        @click="toggleNicklist()"
```
Comment (ItsOnlyBinary) [inline src/components/ContainerHeader.vue]: Having this as a method seems inconsistent, feel this should be reverted and the method removed.

**Pull request merged**